.github/workflows/ci-kafka.yml

```
@@ -34,7 +34,7 @@ jobs:
       run: bash scripts/kafka-integration-test.sh -k
 
     - name: Output Kafka logs
-      run: docker logs kafka
+      run: docker-compose -f docker-compose/kafka-integration-test/v3.yml  logs kafka
```

```suggestion
      run: docker compose -f docker-compose/kafka-integration-test/v3.yml  logs kafka
```
`docker-compose` is deprecated and not always available in the runners